container-images/bamoe-canvas/Containerfile

```diff
@@ -0,0 +1,77 @@
+ARG KIE_SANDBOX_DEV_DEPLOYMENT_BASE_IMAGE_URL_ARG
+ARG KIE_SANDBOX_DEV_DEPLOYMENT_KOGITO_QUARKUS_BLANK_APP_IMAGE_URL_ARG
+ARG KIE_SANDBOX_DEV_DEPLOYMENT_DMN_FORM_WEBAPP_IMAGE_URL_ARG
+
+ENV KIE_SANDBOX_REQUIRE_CUSTOM_COMMIT_MESSAGE=$KIE_SANDBOX_REQUIRE_CUSTOM_COMMIT_MESSAGE_ARG \
+    KIE_SANDBOX_APP_NAME='BAMOE Canvas' \
+    KIE_SANDBOX_DEV_DEPLOYMENT_BASE_IMAGE_URL=$KIE_SANDBOX_DEV_DEPLOYMENT_BASE_IMAGE_URL_ARG \
+    KIE_SANDBOX_DEV_DEPLOYMENT_KOGITO_QUARKUS_BLANK_APP_IMAGE_URL=$KIE_SANDBOX_DEV_DEPLOYMENT_KOGITO_QUARKUS_BLANK_APP_IMAGE_URL_ARG \
+    KIE_SANDBOX_DEV_DEPLOYMENT_DMN_FORM_WEBAPP_IMAGE_URL=$KIE_SANDBOX_DEV_DEPLOYMENT_DMN_FORM_WEBAPP_IMAGE_URL_ARG \
+    KIE_SANDBOX_ACCELERATORS="[ \
+    { \
+        \"name\": \"Quarkus (Full)\", \
+        \"iconUrl\": \"https://github.com/ibm/bamoe-canvas-quarkus-accelerator/raw/$BAMOE_VERSION_FINAL_ARG-quarkus-full/logo.png\", \
+        \"gitRepositoryUrl\": \"https://github.com/ibm/bamoe-canvas-quarkus-accelerator\", \
+        \"gitRepositoryGitRef\": \"$BAMOE_VERSION_FINAL_ARG-quarkus-full\", \
+        \"dmnDestinationFolder\": \"src/main/resources\", \
+        \"bpmnDestinationFolder\": \"src/main/resources\", \
+        \"otherFilesDestinationFolder\": \"src/main/resources\" \
+    }, \
+    { \
+        \"name\": \"Quarkus (DMN)\", \
+        \"iconUrl\": \"https://github.com/ibm/bamoe-canvas-quarkus-accelerator/raw/$BAMOE_VERSION_FINAL_ARG-quarkus-dmn/logo.png\", \
+        \"gitRepositoryUrl\": \"https://github.com/ibm/bamoe-canvas-quarkus-accelerator\", \
+        \"gitRepositoryGitRef\": \"$BAMOE_VERSION_FINAL_ARG-quarkus-dmn\", \
+        \"dmnDestinationFolder\": \"src/main/resources\", \
+        \"bpmnDestinationFolder\": \"src/main/resources\", \
+        \"otherFilesDestinationFolder\": \"src/main/resources\" \
+    } \
+]" \
+    KIE_SANDBOX_EDITORS='[ \
+    { \
+        "extension": "bpmn", \
+        "filePathGlob": "**/*.bpmn?(2)", \
+        "editor": { \
+            "resourcesPathPrefix": "gwt-editors/bpmn", \
+            "path": "bpmn-envelope.html" \
+        }, \
+        "card": { \
+            "title": "Workflow", \
+            "description": "BPMN files are used to generate business workflows." \
+        } \
+    }, \
+    { \
+        "extension": "dmn", \
+        "filePathGlob": "**/*.dmn", \
+        "editor": { \
+            "resourcesPathPrefix": "", \
+            "path": "new-dmn-editor-envelope.html" \
+        }, \
+        "card": { \
+            "title": "Decision", \
+            "description": "DMN files are used to generate decision models" \
+        } \
+    } \
+]'
+
+# Removing PMML assets, as they're not part of BAMOE.
+# Disabled until https://github.com/apache/incubator-kie-issues/issues/327 is resolved.
+#RUN rm /var/www/html/samples/Sample.pmml && \
+#    rm /var/www/html/pmml-envelope.html && \
+#    rm /var/www/html/pmml-envelope.js
+
+# Custom branding.
+# These are put in place using the script BuildBamoeImage.java
```

Comment:
another reference for BuildBamoeImage.java

Reply:
Fixed.